backend/modules/simulation/core/orchestrator.py

```
@@ -0,0 +1,699 @@
+            return []
+        
+        try:
+            db = SessionLocal()
+            try:
+                personas = db.query(SimulationPersona).join(
+                    scene_personas_table, SimulationPersona.id == scene_personas_table.c.persona_id
+                ).filter(scene_personas_table.c.scene_id == scene_id).all()
+                return personas
+            finally:
+                db.close()
+        except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)